README.md

```diff
@@ -23,3 +23,4 @@ Give a ⭐️ if you like this project!
 
 Acknowledgment:
 I thank Microverse for the materials given to understand the git workflow and use of linter and figma
+
```

Comment:
kindly make sure your README file looks professional. remember that this file is very important because it describes your project and therefore it would be better if you complete all the steps so that your Readme is professional